package.json

```
@@ -2,7 +2,7 @@
   "name": "microsoft-cognitiveservices-speech-sdk",
   "author": "Microsoft Corporation",
   "homepage": "https://docs.microsoft.com/azure/cognitive-services/speech-service/",
-  "version": "1.3.0-alpha.0.1",
+  "version": "1.4.1",
```

Please leave version as is, version is fixed up from our release branches:

```suggestion
  "version": "1.3.0-alpha.0.1",
```

---

Done. See the latest (squashed) commit.
You should also consider dropping `gulp-change` from devDependencies, as it is not used.

---

thanks!